### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Wakahisa**
Basic requirements added to Readme

### CODE REVIEW

**Comment by ericpak** [comment]
what is this shit?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
 # line-bot
 Line bot for our random features
+
+## Instructions to get started:
+
+Requirements: Python 3.6+, pipenv, Node
```
Comment (GiantRobato) [inline README.md]: This should be a bullet list

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+
+To get the project running, you will need to install these dependencies:
+
+[Python 3.6+](https://www.python.org/downloads/)
+
+## If you don't have pipenv, run this command
+
+pip install pipenv
```
Comment (GiantRobato) [inline README.md:17]: this should be code block

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+
+[requires]
+python_version = "3.7"
```
Comment (GiantRobato) [inline Pipfile]: this is not the version we're using

**Review by GiantRobato — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Wakahisa) [reply README.md:17]: Completed

File: Pipfile [hunk 4]
```diff
@@ -6,7 +6,10 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
+<<<<<<< HEAD
```
Comment (GiantRobato) [inline Pipfile]: can you fix this merge conflict?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,25 @@
...
+## Instructions to get started:
+
+Requirements: 
+* Python 3.6+ 
+* pipenv
+* Node
+
+To get the project running, you will need to install these dependencies:
+
+[Python 3.6+](https://www.python.org/downloads/)
```
Comment (GiantRobato) [inline README.md]: This is already stated in the requirements

File: README.md [hunk 6]
```diff
@@ -1,2 +1,25 @@
 # line-bot
 Line bot for our random features
+
+## Instructions to get started:
+
+Requirements: 
+* Python 3.6+
```
Comment (GiantRobato) [inline README.md]: Add link here

File: README.md [hunk 7]
```diff
@@ -1,2 +1,25 @@
 # line-bot
 Line bot for our random features
+
+## Instructions to get started:
+
+Requirements: 
+* Python 3.6+ 
+* pipenv
```
Comment (GiantRobato) [inline README.md]: you should put python package requirements and node requirements in a separate section.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,25 @@
...
+
+[Python 3.6+](https://www.python.org/downloads/)
+
+## If you don't have pipenv, run this command
+```
+pip install pipenv
+```
+## Run this command in the root of the project folder
+
+* pipenv shell
```
Comment (GiantRobato) [inline README.md]: needs to be in a code section

**Review by GiantRobato — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (Wakahisa) [reply README.md]: Change made

**Review by GiantRobato — APPROVED** [review]

**Pull request merged**